wandb/proto/wandb_internal.proto

```diff
@@ -469,10 +469,17 @@ message GetSummaryResponse {
  */
 message StatusRequest {
   bool check_stop_req = 1;
+  bool check_retries = 2;
```

Comment:
I think this field name could be a little more generic. We could choose to return any kind of network status messages through here.